### ISSUE TITLE
refactor(plugin-consortium-manual): move plugin logic from endpoint classes to plugin class

### ISSUE THREAD

**Issue by petermetz** [hyperledger/cactus#429]
### Description

Just move the code out of the endpoint classes into the plugin class to have the separation of concerns best practice/programming pattern enforced: The endpoint class is the joint between the raw HTTP requests and the plugin classes and is only responsible for validating that the requests are compliant with what is being described in the OpenAPI specs.
So because of the above they should not contain the actual logic of the plugin because that belongs in the plugin.

### Acceptance Criteria
1. `packages/cactus-plugin-consortium-manual/src/main/typescript/consortium/get-node-jws-endpoint-v1.ts` has the logic moved to the plugin class at `./packages/cactus-plugin-consortium-manual/src/main/typescript/plugin-consortium-manual.ts`. The constructor options argument of the endpoint takes the plugin as an input (if not already) so that it can invoke the relevant method of the plugin in the place of the code that is being moved there
    ```typescript
    public async createJws(): Promise<JWSGeneral> {
      const fnTag = "GetNodeJwsEndpoint#createJws()";
      const { keyPairPem, consortium } = this.options;
      try {
        const keyPair = JWK.asKey(keyPairPem);
        const payload = jsonStableStringify({ consortium });
        const _protected = {
          iat: Date.now(),
          jti: uuid.v4(),
          iss: "Hyperledger Cactus",
        };
        // TODO: double check if this casting is safe (it is supposed to be)
        return JWS.sign.general(payload, keyPair, _protected) as JWSGeneral;
      } catch (ex) {
        throw new Error(`${fnTag} ${ex.stack}`);
      }
  }
    ```
 
2. Same as 1) but for `packages/cactus-plugin-consortium-manual/src/main/typescript/consortium/get-consortium-jws-endpoint-v1.ts`
    ```typescript
    const nodes2d = this.options.consortium.members?.map((m) => m.nodes);
    const nodes = flatten(nodes2d);
    const requests: Promise<AxiosResponse<GetNodeJwsResponse>>[] = nodes
      .map((cnm) => cnm.nodeApiHost)
      .map((host) => new Configuration({ basePath: host }))
      .map((configuration) => new DefaultApi(configuration))
      .map((apiClient) =>
        apiClient.apiV1PluginsHyperledgerCactusPluginConsortiumManualNodeJwsGet()
      );
    const responses = await Promise.all(requests);
  
    const signatures: JWSRecipient[] = [];
  
    responses
      .map((apiResponse) => apiResponse.data)
      .map((getNodeJwsResponse) => getNodeJwsResponse.jws)
      .forEach((aJws: JWSGeneral) =>
        aJws.signatures.forEach((signature) => signatures.push(signature))
      );
  
    const [response] = responses;
    const jws = response.data.jws;
    jws.signatures = signatures;
  
    const body = { jws };
    ```
3. The new methods on the plugin class at `./plugin at `./packages/cactus-plugin-consortium-manual/src/main/typescript/plugin-consortium-manual.ts` to have a single input parameter in the form of the request body and the output needs to be the response body object (both of these types should be defined by the open API spec of the `core-api` package, there's another issue open specifically for this: #427 

Depends on #427 

cc: @sfuji822 @takeutak @hartm @jonathan-m-hamilton

**Comment by jordigiam** [comment]
Working on this
